src/utilities/FormManager.ts

```diff
@@ -267,7 +267,7 @@ class FormManager {
     this.notifyUserAboutErrors(backendErrors.errorMessages);
   }
 
-  public async onFinish(_values: any) {
+  public async onFinish(_values?: any) {
```

Comment:
Does this work? Actually asking.

```suggestion
  public async onFinish() {
```